www/statusbar.js

```diff
@@ -153,6 +156,21 @@ var addStatusBarDataElement = function(){
 }
 
 
+var injectViewportMetaTag = function(){
+
+    if(device.platform.toLowerCase().indexOf(IOS_PLATFORM) == 0 && device.version.split(".")[0] >= IOS_11_VERSION){
```

Comment:
Is this device object provided by the Device plugin?

Reply:
Yes. https://cordova.apache.org/docs/en/latest/reference/cordova-plugin-device/